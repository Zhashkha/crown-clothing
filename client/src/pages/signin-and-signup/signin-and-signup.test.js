import React from "react";
import { shallow } from "enzyme";

import SigninAndSignupPage from "./signin-and-signup.component";

it("test SigninAndSignupPage snapshot", () => {
  expect(shallow(<SigninAndSignupPage />)).toMatchSnapshot();
});
