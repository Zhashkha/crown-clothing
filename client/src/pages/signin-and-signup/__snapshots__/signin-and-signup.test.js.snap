// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test SigninAndSignupPage snapshot 1`] = `ShallowWrapper {}`;
